barsys/templates/email_dutch/bank_account_details.html.html

```diff
@@ -17,5 +17,11 @@
         <th>Betalingskenmerk</th>
         <td>{{ pybarsys_preferences.EMAIL.BANK_ACCOUNT_PAYMENT_REFERENCE }} {{ recipient.display_name|clean_str }}</td>
     </tr>
+    {% if invoice.comment != "" %}
```

Comment:
Can you also move the comment stuff in the emails to here? The comment, at least in general, does not have to do anything with the bank account. This will then lead to other files needing to be touched instead of the ones for bank account details, and you'll need to add a new table in the non-plaintext ones.

![image](https://user-images.githubusercontent.com/1850504/95599120-f6d8d900-0a50-11eb-82d4-50fd72aed11e.png)

Relevant for all three mail languages

Reply:
moved it